lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -420,21 +420,25 @@ bool isAttributeEditAllowed(const PXR_NS::UsdAttribute& attr, std::string* errMs
     return true;
 }
 
-bool isAttributeEditAllowed(const PXR_NS::UsdPrim& prim, const std::string& tokenName)
+bool isAttributeEditAllowed(const UsdPrim& prim, const TfToken& attrName)
 {
-    std::string errMsg;
+    if (prim.IsA<UsdGeomXformable>()) {
```

Comment:
As you have it now, the code is UsdGeomXformable centric, and it doesn't need to be --- we currently return true if it's not a UsdGeomXformable.  Seems to me we could have something like:

UsdGeomXformable xformable(prim);
if (xformable) { // bool operator on schema returns true if valid
    if (UsdGeomXformOp::IsXformOp(attrName) {
        // Check xformOpOrder
    }
}
// Then check the attribute itself.

Reply:
@ppt-adsk 0b42a42b6ad4485c780128192e77f0aa35aed0b9